docs/userguide/mpi_apps.rst

```diff
@@ -1,6 +1,14 @@
 MPI Apps
 ========
 
+.. note::
```

Comment:
I believe this change is unintentional because this notice was removed in the past couple of days.